app/templates/events/trackVolunteers.html

```diff
@@ -37,6 +37,9 @@ <h1 class="text-center">{{event.name}}</h1>
 <div class="flex-container" style="padding: 5px;">
   <div class='form-group d-inline-flex'>
     <a class="btn btn-success" role="button" href="#" data-bs-toggle="modal" data-bs-target="#addVolunteerModal" style="margin:5px;">Add volunteer</a>
+    {% if isRecurring or isNotFirstRecurringEvent%}
```

Comment:
These are properties of the event, and aren't passed in separately from the controller.

Go ahead and change `isPastEvent` as well